packages/network/src/api/image/requests.ts

```diff
@@ -365,8 +380,10 @@ export async function addImage(
     const image = mapApiImage(
       body,
       options.inspectionId,
+      config.thumbnailDomain,
       (options as AddBeautyShotImageOptions).compliance,
     );
+    fetchThumbnail(options, image);
```

Comment:
Properly handle the error case for this promise call (we try to avoid unhandled promise rejections) :

fetchThumbnail(options, image).catch(console.error);